src/Penrose/Functions.hs

```diff
@@ -815,8 +789,9 @@ sampleFunctionArea [GPI domain, GPI range, Val (FloatV xFrac), Val (FloatV yFrac
              y_offset = (0, dy)
              pt_midright = midpoint pt_tr pt_br -: x_offset +: y_offset
              pt_midleft = midpoint pt_bl pt_tl +: x_offset +: y_offset
-             right_curve = interpolateFn [pt_tr, pt_midright, pt_br]
-             left_curve = interpolateFn [pt_bl, pt_midleft, pt_tl]
+             -- FIXME: take k as arg
```

Comment:
Did you want to come back and fix this now @wodeni?

Reply:
Nope. Turns out that changing `k` doesn't fundamentally solve the shape-going-outta-bbox issue. So I didn't end up doing it. Removing the comment.